zulia-server/src/main/java/io/zulia/server/cmd/zuliad/ZuliaDConfig.java

```diff
@@ -50,7 +52,13 @@ public ZuliaDConfig(String config) throws IOException {
 
 		LOG.info("Loading config <" + config + ">");
 
-		zuliaConfig = GSON.fromJson(new FileReader(config), ZuliaConfig.class);
+		if (config.endsWith("json") || config.endsWith("properties")) {
+			zuliaConfig = GSON.fromJson(new FileReader(config), ZuliaConfig.class);
+		}
+		else if (config.endsWith("yml") || config.endsWith("yaml")) {
+			Yaml yaml = new Yaml(new Constructor(ZuliaConfig.class));
+			zuliaConfig = yaml.load(new FileReader(config));
```

Comment:
Do we need to close the file reader (in json and yaml)?

Reply:
I'm not seeing a warning, so maybe not?